master/usage/routereflector/calico-routereflector.md

```diff
@@ -0,0 +1,277 @@
+---
+title: 'Calico BIRD Route Reflector container'
+---
+
+This guide discusses the calico/routereflector image: a container image that
```

Comment:
One thing I'm concerned about in this guide (as well as the the existing route reflector doc) is that it doesn't make clear when you should use a route reflector.

For someone who doesn't know if they need a route reflector or not (they probably don't) this could be a really scary doc.

Can we add a bit at the top which says something like "Most deployments don't require a route reflector.  You may be interested in adding a route reflector to your deployment if ... " or similar?